deployment/prerequisites.ts

```diff
@@ -55,21 +50,27 @@ export class PrerequisiteStack extends Stack {
       }
     );
 
-    new apigateway.CfnAccount(
-      this,
-      "ApiGatewayRestApiAccount",
-      {
-        cloudWatchRoleArn: cloudWatchRole.roleArn,
-      }
-    );
+    new apigateway.CfnAccount(this, "ApiGatewayRestApiAccount", {
+      cloudWatchRoleArn: cloudWatchRole.roleArn,
+    });
   }
 }
 
 async function main() {
   const app = new App({
-    defaultStackSynthesizer: new DefaultStackSynthesizer(
-      JSON.parse((await getSecret("cdkSynthesizerConfig"))!)
-    ),
+    defaultStackSynthesizer: new DefaultStackSynthesizer({
```

Comment:
potentially we could put this in a json file or something and then import in the 2 spots. Also not a huge deal breaker.

Reply:
Yeah, if you'd like.  I wish the whole thing didn't exist at all.